airflow/contrib/utils/bluecore_bq_gcs_task_handler.py

```diff
@@ -95,6 +97,17 @@ def _read(self, ti, try_number):
                     bq_table_name = "{}.gke_logs.{}_*".format(data_project_name, container_name)
                     tables.append((bq_table_name, pod_id))
 
+            if not tables:
+                message = "No pods were found running for this task, and the task's output has not been written to " \
+                          "GCS.  If refreshing this page doesn't present a log, it's possible that this task failed " \
+                          "to spawn a kubernetes job.\n\n"
+                message += json.dumps(tables) + "\n\n"
+                try:
+                    message += FileTaskHandler._read(self, ti, try_number)
+                except:
+                    message += "Failed to read airflow worker log!  Stack trace:\n{}".format(traceback.format_exc())
+                return message
+
```

Comment:
These changes are helpful when developing airflow in the event that your changes prevent pods from coming up at all.